### ISSUE TITLE
Confirm Auto-Translation of Gitcoin to Korean

### ISSUE THREAD

**Issue by ceresstation** [gitcoinco/web#1680]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)

As a user I want to fulfill / fund bounties in my native language.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: For maximal inclusion :)

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)

To start, we'll probably want to generate the base (empty) translation files as in https://github.com/gitcoinco/web/pull/971

There are two .po files that will need modified for each language we wish to support. 

`django.po` and `djangojs.po`

You can see how some of them are populated [in 971](https://github.com/gitcoinco/web/pull/971/files).

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

The auto-translation of the site to Korean is confirmed :)

**Comment by owocki** [comment]
@ceresstation is there a specific need from korean coming from the market?

**Comment by ceresstation** [comment]
@owocki Just a starting point, waiting for Mark to make a few changes to #971 and then bountying this + adding similar issues for other translations in parallel.

**Comment by owocki** [comment]
my question was more 'why is korean strategically important / why should it be first?'

**Comment by ceresstation** [comment]
@owocki Each translation is just a separate task to be done in parallel, this was just a test issue spec'd out while Mark completed the autotranslate implementation.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.1 ETH (46.66 USD @ $466.59/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1680/873).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/1680/873).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $30,457.55 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by kaushalvivek** [comment]
Hey @ceresstation, I would like to work on this.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 2 months, 1 week from now.
Please review their action plans below:


1. [aashutoshrathi](https://gitcoin.co/profile/aashutoshrathi) has started work. 
    
    Create ko/django.po and ko/djangojs.po for auto translation

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1680/873).

**Comment by gitcoinbot** [comment]
@aashutoshrathi Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=100)

**Comment by aashutoshrathi** [comment]
@ceresstation This issue has been already solved the django. po and djangojs.po in folder `ko` are already made and populated.
What else is to be done in this issue.

**Comment by gitcoinbot** [comment]
@aashutoshrathi Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=100)

**Comment by thelostone-mc** [comment]
Ah @aashutoshrathi ! Sorry about that 
I'm snoozing the bot -> @ceresstation is traveling! Mind waiting for a few days ?

**Comment by gitcoinbot** [comment]
@aashutoshrathi Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1680/873?snooze=100)

**Comment by ceresstation** [comment]
Sorry about that @aashutoshrathi, paying out the bounty now!

**Comment by aashutoshrathi** [comment]
@ceresstation Also close that bounty as it is already completed.
And please take a look at #2003

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.10000  ETH (22.82 USD @ $228.23/ETH) has been granted to @aashutoshrathi for this issue from @ceresstation. ⚡️ 

Nice work @aashutoshrathi! To redeem your tip, login to Gitcoin at https://gitcoin.co/explorer and select 'Claim Tip' from dropdown menu in the top right, or check your email for a link to the tip redemption page. 

 * $21552.67 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 0.1 ETH (22.82 USD @ $228.23/ETH) (plus a crowdfund of 0.01 ETH worth $4.53) attached to this issue has been approved & issued.__  

 * Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1680/873)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $21,552.67 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by mbeacom** [comment]
@ceresstation @aashutoshrathi were there any corrections to the Korean translations? The only thing I'm seeing is the PR for adding Japanese. 🤔

**Comment by aashutoshrathi** [comment]
Korean one was already done, I completed Japanese one.

**Comment by mbeacom** [comment]
@ceresstation @thelostone-mc  Sounds like we still need to verify the Korean translation.
